addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -139,7 +152,7 @@ def test_operator_livechat_username(self):
             'email': 'michel@example.com',
             'livechat_username': 'Michel at your service',
         })
-        channel_info = self.livechat_channel.with_user(public_user).sudo()._open_livechat_discuss_channel(anonymous_name='whatever')
+        channel_info = self.make_rpc('/im_livechat/get_session', {'anonymous_name': 'whatever', 'channel_id': self.livechat_channel.id})
```

Comment:
Could make actually using the RPC in tests as prep too I feel.
This would even better guarantee the results are the same before/after this PR.